dart/lib/src/environment_variables.dart

```diff
@@ -0,0 +1,33 @@
+/// Reads environment variables from the system.
+/// In an Flutter environment these can be set via
+/// `flutter build --dart-define=VARIABLE_NAME=VARIABLE_VALUE`.
+class EnvironmentVariables {
+  static const _sentryEnvironment = 'SENTRY_ENVIRONMENT';
+  static const _sentryDsn = 'SENTRY_DSN';
+  static const _sentryRelease = 'SENTRY_RELEASE';
+  static const _sentryDist = 'SENTRY_DSN';
+
+  /// `SENTRY_ENVIRONMENT`
+  /// See [SentryOptions.environment]
+  String? get environment => const bool.hasEnvironment(_sentryEnvironment)
```

Comment:
That's a named constructor so it can't return null.
So it wouldn't throw but it would return an empty string (`''`) if the environment variable does not exist.

Reply:
maybe `fromEnvironment` passing `defaultValue=null` if possible would do then

Reply:
You can't pass null because it only accepts non-null strings (because null safety), so yeah it is already correct as is.